code/modules/mob/living/carbon/human/species/station/diona/diona.dm

```diff
@@ -25,7 +25,8 @@
 	siemens_coefficient = 0.3
 	eyes = "blank_eyes"
 	show_ssd = "completely quiescent"
-	num_alternate_languages = 1
+	num_alternate_languages = 2
```

Comment:
Gonna point out that this will give dionae access to three languages at maximum, since they start with rootson by default, against everyone's else two.

Reply:
Unless i'm misunderstanding this, dionae start with basic, rootsong, and a free slot. This gives them another free slot. It doesn't make much sense they are limited this much compared to some other species, with shells getting three free slots and skrell four.